files/build_templates/sonic_debian_extension.j2

```diff
@@ -286,6 +286,12 @@ sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.sh $FILESYSTEM_ROOT/usr/bin/
 sudo cp $IMAGE_CONFIGS/interfaces/*.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES/
 echo "interfaces-config.service" | sudo tee -a $GENERATED_SERVICE_FILE
 
+# Copy CoPP configuration files and templates
+sudo cp $IMAGE_CONFIGS/copp/copp-config.service $FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM
+sudo cp $IMAGE_CONFIGS/copp/copp-config.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp $IMAGE_CONFIGS/copp/copp_cfg.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES/
+echo "copp-config.service" | sudo tee -a $GENERATED_SERVICE_FILE
```

Comment:
Instead of having a service to generate copp_cfg.json, can we do it as part of docker-orchagent/docker-init.sh?

Reply:
The file is in the host and not inside the docker and hence we need it to be created through a one shot task.

Reply:
ok